python/src/aac/plugins/gen_plugin/__init__.py

```diff
@@ -116,6 +116,8 @@ def run_gen_project(
     else:
         result.add_messages(gen_project_result.messages)
 
+    if not output or len(output) == 0:
```

Comment:
This check and default over to current location handling fixed the manual testing of not providing an output location manually and works as expected when providing an output location